src/components/CreateAnalysis/CreateAnalysisWrapper.tsx

```diff
@@ -14,18 +14,25 @@ const CreateAnalysisWrapper = () => {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const history = useHistory();
 
-  const submitAnalysis = () => {
-    const imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages, selectedStudyUIDs);
-    if (imagesSelected.length <= 0) {
-      setIsModalOpen(true);
-      return;
+  const submitAnalysis = (XrayModel: string | void, CTModel: string | void) => {
```

Comment:
Did you mean that `XrayModel` and `CTModel` can be `undefined`? In that case, I would suggest 
```suggestion
  const submitAnalysis = (XrayModel?: string, CTModel?: string) => {
```